internal/annotations/scorecard/scorecard.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package scorecard
+
+import (
+	"path/filepath"
+)
+
+// Static bundle annotation values.
+const (
+	mediaTypeV1 = "scorecard+v1"
+)
+
+// Bundle annotation keys.
+// TODO: version these variables somehow (either in name or in subpackage).
```

Comment:
I could not get what would be required to get done in the TODO. Could we describe it better? 
Can we create an issue with what needs to be done and link here?

Reply:
This is a bundle spec thing, and internal to the SDK (users will never see this), so its not worth creating an issue. I'll explain more in the TODO.